src/renderer/components/DuplicateItem.tsx

```
@@ -38,10 +44,13 @@ const DuplicateItem: React.FC<DuplicateItemProps> = memo(({
     let recommended = duplicate.tracks[0];
 
     if (resolutionStrategy === 'keep-highest-quality') {
+      const qualityScore = (t: any) => (t.bitrate || 0) + (t.size || 0) / 1000000;
       recommended = duplicate.tracks.reduce((best: any, current: any) => {
-        const bestScore = (best.bitrate || 0) + (best.size || 0) / 1000000;
-        const currentScore = (current.bitrate || 0) + (current.size || 0) / 1000000;
-        return currentScore > bestScore ? current : best;
+        const bestLossless = isLossless(best.location || '');
+        const currentLossless = isLossless(current.location || '');
+        if (currentLossless && !bestLossless) return current;
```

ESLint

[error] Expected { after 'if' condition.

(curly)